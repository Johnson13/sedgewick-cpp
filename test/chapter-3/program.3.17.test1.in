Lorem
ipsum
dolor
sit
amet,
consectetur
adipiscing
elit.
Aenean
porta
augue
sed
purus
lobortis
iaculis.
Mauris
commodo
id
nunc
ut
pretium.
Nulla
laoreet
est
in
nibh
consequat,
id
vehicula
dui
sodales.
Maecenas
ut
odio
ornare,
iaculis
nunc
in,
feugiat
nisi.
Cras
molestie
finibus
tempor.
Integer
et
urna
a
neque
tincidunt.
