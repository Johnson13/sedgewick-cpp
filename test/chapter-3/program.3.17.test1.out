Aenean
Cras
Integer
Lorem
Maecenas
Mauris
Nulla
a
adipiscing
amet,
augue
commodo
consectetur
consequat,
dolor
dui
elit.
est
et
feugiat
finibus
iaculis
iaculis.
id
id
in
in,
ipsum
laoreet
lobortis
molestie
neque
nibh
nisi.
nunc
nunc
odio
ornare,
porta
pretium.
purus
sed
sit
sodales.
tempor.
tincidunt.
urna
ut
ut
vehicula
